EIPS/eip-1363.md

```diff
@@ -184,23 +186,33 @@ interface ERC1363Spender {
 ``` 
 
 ## Rationale
-There are many proposed uses of Ethereum smart contracts that can accept ERC-20 payments (i.e. to create a token payable crowdsale, selling services for tokens, paying invoices, making subscriptions, use them for a specific utility and many other purposes). 
+There are many proposed uses of Ethereum smart contracts that can accept [ERC-20](./eip-20.md) payments. 
+
+Examples could be 
+* to create a token payable crowdsale
+* selling services for tokens 
+* paying invoices
+* making subscriptions
+
+For these reasons it was named as **"Payable Token"**.
+
+Anyway you can use it for specific utilities or for any other purposes who require the execution of a callback after a transfer or approval received.
 
-This proposal wants to make token payments easier and working without the use of any other listener. It also allows to make a callback after a transfer in a single transaction.
+The choice to use `transferAndCall`, `transferFromAndCall` and `approveAndCall` derives from the [ERC-20](./eip-20.md) naming. They want to highlight that they have the same behaviours of `transfer`, `transferFrom` and `approve` with the addition of a callback on receiver or spender.
 
 ## Backwards Compatibility
-This proposal is inspired also by ERC-223 and ERC-677 but it uses the ERC-721 approach so it doesn't override the ERC-20 `transfer` and `transferFrom` methods and defines the interfaces IDs to be implemented maintaining the ERC-20 backwards compatibility.  
+This proposal has been inspired also by [ERC-223](https://github.com/ethereum/EIPs/issues/223) and [ERC-677](https://github.com/ethereum/EIPs/issues/677) but it uses the [ERC-721](./eip-721.md) approach, so it doesn't override the [ERC-20](./eip-20.md) `transfer` and `transferFrom` methods and defines the interfaces IDs to be implemented maintaining the [ERC-20](./eip-20.md) backwards compatibility.  
 
 ## Test Cases
-[https://github.com/vittominacori/erc1363-payable-token/tree/master/test](https://github.com/vittominacori/erc1363-payable-token/tree/master/test)
+The [repository with the reference implementation](https://github.com/vittominacori/erc1363-payable-token) contains all the [tests](https://github.com/vittominacori/erc1363-payable-token/tree/master/test).
 
 ## Implementation
-[https://github.com/vittominacori/erc1363-payable-token/tree/master/contracts/token/ERC1363](https://github.com/vittominacori/erc1363-payable-token/tree/master/contracts/token/ERC1363)
+The GitHub repository [vittominacori/erc1363-payable-token](https://github.com/vittominacori/erc1363-payable-token) contains the reference implementation. The reference implementation is also available via [npm](https://www.npmjs.com/package/erc-payable-token) and can be installed with `npm install erc-payable-token`.
```

Comment:
In case you weren't aware, Implementation and Test Cases sections are not required.  For a token like this, the value add the provide is limited IMO, so don't feel like you *must* get these integrated into this repository.  Instead, you could just remove these sections.  😄

Reply:
Can I add them as references?

Reply:
We generally prefer to avoid that where possible.  The problems we are trying to solve here are:
1. **Advertising**.  People try to use EIPs to advertise their project by linking to their project from the EIP, hoping that this will drive traffic to their project.  By putting everything inside this repository, we don't have to worry about that problem.
2. **Licensing**.  Everything in the EIPs repository is CC0.  If you link out to something that is a different license (i.e., not public domain) it can create licensing problems for people down the road who aren't careful.  For example, if the CC0 licensed EIP links out to a GPL licensed implementation and in the course of reading the EIP the reader clicks through to the GPL code and reads it, there is now grounds for an argument that they have violated copyright when their code ends up looking like the GPL code they saw.

By having everything inside this repository (and thus licensed CC0), or simply left out of the EIP, we can avoid both of these problems.  It is valuable to have example code and test cases available to readers, but we don't need an entire test project or demo project.  The code alone is usually enough, and if someone really wants they can do a quick search online for EIP-1363 to find other people's implementations and test cases.